app/latigo/scheduler/__init__.py

```diff
@@ -158,13 +158,12 @@ def perform_prediction_step(self):
         stats_models_ok = {}
         stats_projects_bad = {}
         stats_models_bad = {}
-        stats_start_time = datetime.datetime.now()
-        prediction_start_time = (
-            datetime.datetime.now() - self.continuous_prediction_delay
-        )
-        prediction_end_time = (
-            prediction_start_time + self.continuous_prediction_interval
-        )
+        now_utc = get_datetime_now_in_utc()
+        stats_start_time = now_utc
+        # 'prediction_start_time' and 'prediction_end_time' should be in UTC time cause Queue will ignore timezone.
+        prediction_start_time_utc = now_utc - self.continuous_prediction_delay
+        prediction_end_time_utc = prediction_start_time_utc + self.continuous_prediction_interval
```

Comment:
```suggestion
        stats_start_time = get_datetime_now_in_utc()
        # 'prediction_start_time' and 'prediction_end_time' should be in UTC time cause Queue will ignore timezone.
        prediction_start_time_utc = stats_start_time - self.continuous_prediction_delay
        prediction_end_time_utc = prediction_start_time_utc + self.continuous_prediction_interval
```
There is no need to have 2 variables at the same place, and that contain the same value.